tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1104,6 +1104,30 @@ def test_log_softmax_invalid(device, a_shape, axis, dtype):
         return chainerx.log_softmax(a, axis=axis)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'expected_dtypes': _expected_float_dtypes_math_functions,
```

Comment:
Great, it looks good to me except that you're only testing floating values. Could you rewrite it as `TestTrueDivide` https://github.com/chainer/chainer/blob/292566ecee1e8616e88c72d8c07947315eaab513/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L686 ?